mockups/mergepath.html

```
@@ -0,0 +1,1444 @@
+
+      <div class="panel">
+        <h2>Gate Rules</h2>
+        <p class="help">What pushes a PR to external review.</p>
+
+        <div style="margin-top: 14px;">
+          <div class="switch-copy" style="margin-bottom: 6px;">
+            <strong>External review threshold</strong>
+            <span>Lines changed at or above this value escalate to Phase 4.</span>
+          </div>
+          <div class="slider-row">
+            <input type="range" id="threshold" min="50" max="1000" step="10" value="300" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)